pytorch/jobs/docker/1.9/py3/Dockerfile.gpu

```diff
@@ -16,10 +16,13 @@ ENV PYTHONIOENCODING=UTF-8
 ENV LANG=C.UTF-8
 ENV LC_ALL=C.UTF-8
 
-ARG PYTHON=python3.7
+
+ARG PYTHON=python3.8
 ARG PYTHON_PIP=python3-pip
 ARG PIP=pip3
-ARG PYTHON_VERSION=3.7.10
+ARG PYTHON_VERSION=3.8.10
+
+RUN apt-key adv --fetch-keys http://developer.download.nvidia.com/compute/cuda/repos/ubuntu1604/x86_64/3bf863cc.pub
 
 RUN apt-get update \
```

Comment:
I'm not sure how much we need all these since we're now inheriting from the DLC container.

Reply:
This was needed for apt-gets